Automattic/jetpack
# Publicize: re-tidy sidebar components

This is a tidying PR. No big changes.
The idea is to be consistent with core components, without tweaking their visibility. No need to do that.

* Uses Panel, PanelBody, and PanelRow to organize the different components of the plugin in the sidebar
* Use label instead of individual string to label twitter section.
* Clean part of the CSS code.

before | after
-----|------
![image](https://user-images.githubusercontent.com/77539/135325982-795432ce-1202-49be-b648-8a01566620fe.png) | ![image](https://user-images.githubusercontent.com/77539/135325860-21dbb16c-ba5a-41c6-9394-87e66a08d284.png)


Fixes https://github.com/Automattic/jetpack/issues/21239

#### Changes proposed in this Pull Request:

* Publicize: slightly re-tidy sidebar components.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Apply these changes
* Confirm the feature is working, as usual, as expected.
* Enable/disable some social media connections to see how the components are visually mounted/unmounted.
